tox.ini

```diff
@@ -12,8 +12,28 @@ deps =
     ansible22: ansible~=2.2
 
 commands =
+    unit: pytest
     flake8: flake8
     pylint: python setup.py lint
     yamllint: python setup.py yamllint
-    unit: nosetests
     generate_validation: python setup.py generate_validation
+
+[pytest]
+testpaths =
+    # TODO(rhcarvalho): enable these after https://github.com/openshift/openshift-ansible/pull/3414 merges.
+    #roles/lib_openshift/src/test/unit
+    #roles/lib_utils/src/test/unit
+    roles/openshift_master_facts/test
```

Comment:
I replaced `testpaths` with `omit = utils`, and then run tox/pytest with `--collect-only`.
The discovery alone can take > 15s.

I think that's too much, considering the time the tests actually run.

But I've found a different problem, it seems that test discovery gets lost because of how we have lots of symlinks. The same pytest run I mention failed with errors that include:

```
...
=================== ERRORS ====================
______________ ERROR collecting  ______________
.tox/py35-ansible22-unit/lib/python3.5/site-packages/_pytest/config.py:325: in _getconftestmodules
    return self._path2confmods[path]
E   KeyError: local('/home/rodolfo/openshift/src/github.com/openshift/openshift-ansible/playbooks/aws/openshift-cluster/roles/openshift_certificate_expiry/examples/playbooks/roles/openshift_certificate_expiry/ex
amples/playbooks/roles/openshift_certificate_expiry/examples/playbooks/roles/openshift_certificate_expiry/examples/playbooks/roles/openshift_certificate_expiry/examples/playbooks/roles/openshift_certificate_expi
ry/examples/playbooks/roles/openshift_certificate_expiry/examples/playbooks/roles/openshift_certificate_expiry/examples/playbooks/roles/openshift_certificate_expiry/examples/playbooks/roles/openshift_certificate
_expiry/examples/playbooks/roles/openshift_certificate_expiry/examples/playbooks/roles/openshift_certificate_expiry/examples/playbooks/roles/openshift_certificate_expiry/examples/playbooks/roles/openshift_certif
icate_expiry/examples/playbooks/roles/openshift_certificate_expiry/examples/playbooks/roles/openshift_certificate_expiry/examples/playbooks/roles/openshift_certificate_expiry/examples/playbooks/roles/openshift_c
ertificate_expiry/examples/playbooks/roles/openshift_master_facts/test')

During handling of the above exception, another exception occurred:
.tox/py35-ansible22-unit/lib/python3.5/site-packages/_pytest/config.py:356: in _importconftest
    return self._conftestpath2mod[conftestpath]
E   KeyError: local('/home/rodolfo/openshift/src/github.com/openshift/openshift-ansible/playbooks/aws/openshift-cluster/roles/openshift_certificate_expiry/examples/playbooks/roles/openshift_certificate_expiry/ex
amples/playbooks/roles/openshift_certificate_expiry/examples/playbooks/roles/openshift_certificate_expiry/examples/playbooks/roles/openshift_certificate_expiry/examples/playbooks/roles/openshift_certificate_expi
ry/examples/playbooks/roles/openshift_certificate_expiry/examples/playbooks/roles/openshift_certificate_expiry/examples/playbooks/roles/openshift_certificate_expiry/examples/playbooks/roles/openshift_certificate
_expiry/examples/playbooks/roles/openshift_certificate_expiry/examples/playbooks/roles/openshift_certificate_expiry/examples/playbooks/roles/openshift_certificate_expiry/examples/playbooks/roles/openshift_certif
icate_expiry/examples/playbooks/roles/openshift_certificate_expiry/examples/playbooks/roles/openshift_certificate_expiry/examples/playbooks/roles/openshift_certificate_expiry/examples/playbooks/roles/openshift_c
ertificate_expiry/examples/playbooks/roles/openshift_master_facts/test/conftest.py')

During handling of the above exception, another exception occurred:
.tox/py35-ansible22-unit/lib/python3.5/site-packages/py/_path/common.py:366: in visit
    for x in Visitor(fil, rec, ignore, bf, sort).gen(self):
.tox/py35-ansible22-unit/lib/python3.5/site-packages/py/_path/common.py:415: in gen
    for p in self.gen(subdir):
.tox/py35-ansible22-unit/lib/python3.5/site-packages/py/_path/common.py:415: in gen
    for p in self.gen(subdir):
...
```

Looks like some sort of infinite recursion... I should go to sleep ;)